### ISSUE TITLE
Compilation Error in macro default__information_schema (macros\cross_db_utils.sql)

### ISSUE THREAD

**Issue by pswaters** [data-mie/dbt-profiler#24]
**Versions**
dbt: 0.19.1
snowflake: 0.19.1
dbt_profiler: 0.1.3

**Steps To Reproduce**

` dbt run-operation print_profile_docs --args '{"relation_name": "shipment__fct_shipment_stop"}'`

Or this

` dbt run-operation print_profile_docs --args '{"relation_name": "shipment__fct_shipment_stop", "schema": "shipment"}'`

I also tried several other combinations of the above arguments and with each profile macro.

**Results**

Encountered an error while running operation: Compilation Error in macro default__information_schema (macros\cross_db_utils.sql)
  'None' has no attribute 'information_schema'

  > in macro information_schema (macros\cross_db_utils.sql)
  > called by macro get_profile (macros\get_profile.sql)
  > called by macro print_profile_docs (macros\print_profile_docs.sql)
  > called by macro default__information_schema (macros\cross_db_utils.sql)

**Comment by stumelius** [comment]
Thanks for reporting this. It looks like the relation variable is None which means that it doesn't exist or you don't have the permissions to read it. I've changed the behavior PR#25 so that the `get_profile` now throws an exception if the relation does not exist or the user is not authorized.

**Comment by stumelius** [comment]
Did you run my_third_model after copy pasting the configs? If not, the profiling fails because there is no such relation in the database. You can also try specifying the schema and database arguments for the print_profile macro.

**Comment by pswaters** [comment]
Yes, I ran the model and checked to see if it was created in the database.  I also passed in the schema variable to print_profile, and that did not help either.  I even turned my computer off and back on again. :) 
It seems like it should work. Before writing this comment, I reverted the code changes, redid my repro steps, and got the same results.
This might be a good opportunity for me to learn how to debug dbt macros.

**Comment by pswaters** [comment]
Thanks! That worked!  I am still curious to know why it worked on some models without passing in the database and schema names.  I let you know if I find the answer.
